### DISCUSSION TITLE
Support APT\PAA direct installation for Ubuntu & Debian

### DISCUSSION THREAD

**Discussion by Kol9yN** [ghostty-org/ghostty#3685] [Feature Requests, Ideas] [answered]
For non pro users and beginners would be great to have possibility to direct installation, without downloading zig and other strange things and make this process more user friendly.

Please, any luck to see Ghossty in APT?
Thank you so much for your work!

**Comment by 00-kat** [comment]
If I remember correctly, Mitchell is not planning to add packages for any Linux distros (or Homebrew on macOS); rather, those will be done by the community.

There was discussion about addition to the Debian repositories in the [#development channel](https://discord.com/channels/1005603569187160125/1005603569711452192) in the [Ghostty Discord server](https://discord.gg/ghostty) though.

**Reply by Kol9yN** [reply]
There Gentoo, Arch, Fedora, Flake, Void already, that why im asked about Deb\Ubu :)

**Comment by JakeWharton** [comment, marked as answer]
> The Ghostty project only officially provides prebuilt binaries for macOS. Other platforms may provide packages for Ghostty, but those packages are not officially maintained by the Ghostty project.

https://ghostty.org/docs/install/binary

People are already working on Debian packaging.
